src/components/common/AppButton.vue

```diff
@@ -28,13 +28,33 @@ export default defineComponent({
       },
     },
     to: {
-      type: String,
+      type: Object || String,
```

Comment:
This is not the proper way to define it, right now it will be `Object` as a result of this 🐛 
You should define them in an array instead:
```suggestion
      type: [Object, String],
```